### PR TITLE
CrayPE fix for axom

### PR DESCRIPTION
**Pull request by lukebroskop**
pass -ef to the cce fortran compiler, fix the build system to use the correct openmp flag for CCE

### CODE REVIEW

File: var/spack/repos/builtin/packages/axom/package.py [hunk 1]
```diff
@@ -371,6 +371,9 @@ def initconfig_package_entries(self):
     def cmake_args(self):
         options = []
 
+        if '%cce' in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/axom/package.py]: Compiler specs are usually checked with `self.spec.satisfied()`.

Comment (tldahlgren) [inline var/spack/repos/builtin/packages/axom/package.py]:
```suggestion
        if self.spec.satisfies('%cce'):
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/conduit/package.py [hunk 2]
```diff
@@ -354,6 +354,8 @@ def hostconfig(self, spec, prefix):
         if cxxflags:
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS", cxxflags))
         fflags = ' '.join(spec.compiler_flags['fflags'])
+        if '%cce' in self.spec:
+            fflags += "-ef"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/conduit/package.py]: IIRC this is not consistent with a change you made in a PR that has been merged.

File: var/spack/repos/builtin/packages/axom/package.py [hunk 3]
```diff
@@ -380,3 +383,9 @@ def cmake_args(self):
             'BUILD_SHARED_LIBS', 'shared'))
 
         return options
+
+    def patch(self):
+        if '%cce' in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/axom/package.py]:
```suggestion
        if self.spec.satisfies('%cce'):
```

File: var/spack/repos/builtin/packages/conduit/package.py [hunk 4]
```diff
@@ -354,6 +354,8 @@ def hostconfig(self, spec, prefix):
         if cxxflags:
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS", cxxflags))
         fflags = ' '.join(spec.compiler_flags['fflags'])
+        if '%cce' in self.spec:
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/conduit/package.py]:
```suggestion
        if self.spec.satisfies('%cce'):
```

File: var/spack/repos/builtin/packages/conduit/package.py (hunk 2 above)
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/conduit/package.py]:
```suggestion
            fflags += " -ef"
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/conduit/package.py (hunk 2 above)
Comment (lukebroskop) [reply var/spack/repos/builtin/packages/conduit/package.py]: @tldahlgren Do you still want this addressed? I believe fixed this in a subsequent commit?

**Review by tldahlgren — APPROVED** [review]
LGTM

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**